docs/docs/examples/bamboo/REMOTE_AGENTS.md

```diff
@@ -8,23 +8,64 @@ You can learn more details about remote agents on the [official documentation pa
 
 !!!warning "Bamboo server prerequisites"
 
-    * Your Bamboo server instance must be a fully licensed Bamboo Data Center instance
-    * The bamboo server instance must be fully configured
+    * The Bamboo server instance must use a valid Bamboo Data Center instance license and be fully configured
     * The Bamboo server instance must have `security token verification` **enabled**
     * The Bamboo server instance must have `remote agent authentication` **disabled**
 
-## Installation
+## Deployment
 
 Steps required for deploying a remote agent
 
-### Overview
-
 1. Configure Bamboo server for remote agent support
 2. Deploy agent
 
-### Steps
+### 1. Configure Bamboo server
+
+There are 2 approaches for doing this:
+
+* Automatically when deploying Bamboo server
```

Comment:
@nanux @tarka considering that `8.1` of Bamboo is when this will be officially supported (10th Dec) should we hold of on merging these changes until after that date? My thinking is that the docs have been caveated with the fact that Bamboo support is under active dev so maybe its okay to merge?

Reply:
Yeah, this seems fine to me.